public/theme/boltcms-2021/page.twig

```diff
@@ -0,0 +1,35 @@
+{% extends 'partials/_master.twig' %}
+
+{% block main_container %}
+    <main>
+        <div class="subheader">
+            <h1>
+                {{ record|title }}
+            </h1>
+            <p>
+                {{ record.subtitle }}
+            </p>
+        </div>
+        {% if record.image is not empty %}
+            <img class="fluid" src="{{ record.image|thumbnail(800, 600) }}" alt="{{ record.image.alt|default }}">
+        {% endif %}
+
+        {# Ugly fix because right now allow_twig does not work in article. #}
+        {% if '_include_github_contributors' in record.content|raw %}
+            {% set contributors %}
+               {% include 'partials/_contributors.twig'%}
+            {% endset %}
+
+            {{ record.content|replace({'_include_github_contributors': contributors})|raw }}
+        {% else %}
+            {{ record.content }}
+        {% endif %}
+
+        {{ boltform(record.slug, warn=false) }}
+
+        {% if record.slug == 'submit-showcase' and app.request.method == 'POST' %}
+            {% do redirect('/showcase-thank-you') %}
+        {% endif %}
+        {# </div> #}
```

Comment:
Let's delete commented out code, if it's not necessary

Reply:
Check